hack/release/common.sh

```diff
@@ -127,13 +127,17 @@ publishHelmChart() {
     RELEASE_REPO=$3
     HELM_CHART_VERSION=$(helmChartVersion "$RELEASE_VERSION")
     HELM_CHART_FILE_NAME="${CHART_NAME}-${HELM_CHART_VERSION}.tgz"
+    AH_CONFIG_FILE_NAME="${CHART_NAME}/artifacthub-repo.yaml"
 
     cd charts
+    [[ -s "${AH_CONFIG_FILE_NAME}" ]] && oras push "${RELEASE_REPO}:artifacthub.io" --config /dev/null:application/vnd.cncf.artifacthub.config.v1+yaml "${AH_CONFIG_FILE_NAME}:application/vnd.cncf.artifacthub.repository-metadata.layer.v1.yaml"
     helm dependency update "${CHART_NAME}"
     helm lint "${CHART_NAME}"
-    helm package "${CHART_NAME}" --version "$HELM_CHART_VERSION"
+    helm package "${CHART_NAME}" --version "${HELM_CHART_VERSION}"
     helm push "${HELM_CHART_FILE_NAME}" "oci://${RELEASE_REPO}"
     rm "${HELM_CHART_FILE_NAME}"
+    digest="$(crane digest "${RELEASE_REPO}:${HELM_CHART_VERSION}")"
+    cosign sign --yes "${RELEASE_REPO}:${HELM_CHART_VERSION}@${digest}"
```

Comment:
Nice! I've been wanting to do this for a while! Can we break this out into a separate PR as well? Super excited that we will be cosigning these now!

As an aside, do you think there is value in adding a note on valdiating the cosign signature as part of the helm chart install in the docs? Or at least to mention it somewhere.

Reply:
Yes, I think this would be a useful addition, maybe an optional sub-step?

Reply:
> maybe an optional sub-step

Yeah, definitely an optional sub-step, not required at all! I was mostly just thinking if you were going to be opening a PR to be handling the cosigning that it might be nice to add the verification piece there with that change